test/helpers/fixture_set.rb

```diff
@@ -0,0 +1,76 @@
+require 'helpers/kubeclient_helper'
+
+module FixtureSetAssertions
+  class FixtureSet
```

Comment:
This name doesn't seem quite like what I'd expect.. this seems more like fairly generic resource helpers, not that tied to fixtures as the `FixtureDeployHelper`